Add built-in monitoring (#9774)
pingcap/docs: tidb-cloud/monitor-tidb-cluster.md

## Monitoring metrics

In TiDB Cloud, you can view the commonly used metrics of a cluster from the following pages:

- Cluster overview page
- Cluster monitoring page

### Metrics on the cluster overview page

The cluster overview page provides general metrics of a cluster, including Total QPS, Latency, Connections, TiFlash Request QPS, TiFlash Request Duration, TiFlash Storage Size, TiKV Storage Size, TiDB CPU, TiKV CPU, TiKV IO Read, and TiKV IO Write.

> To open the cluster overview page, take the following steps:

Reviewer: ```suggestion
To view metrics on the cluster overview page, take the following steps:
```

1. Navigate to the **Active Clusters** page.

2. Click the name of a cluster to go to its cluster overview page.

### Metrics on the cluster monitoring page

> The cluster monitoring page provides a full standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.

Reviewer: ```suggestion
The cluster monitoring page provides a full set of standard metrics of a cluster. By viewing these metrics, you can easily identify performance issues and determine whether your current database deployment meets your requirements.
```

> 

Reviewer: ```suggestion

> **Note:**
>
> Currently, the Monitoring page is unavailable for [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier).

```

> To open the cluster monitoring page, take the following steps:

Reviewer: ```suggestion
To view metrics on the cluster monitoring page, take the following steps:
```

> 1. Navigate to the **Diagnostics** tab of a cluster.

Reviewer 2: ```suggestion
1. Navigate to the **Diagnosis** tab of a cluster.
```
